File: java/src/model/referi/Referi.java
package model.referi;

import model.batalla.Batalla;

public class Referi {

   private Batalla batalla;

   public Referi(Batalla batalla) {
      this.batalla = batalla;
   }

   public void gestionarBatalla(){

      getBatalla().comenzarBatalla();
   }

   public Batalla getBatalla() {
      return batalla;
   }

   public void setBatalla(Batalla batalla) {
      this.batalla = batalla;
   }
}

File: python/model/referi/referi.py
from model.batalla.batalla import Batalla

class Referi ():
    def __init__(self, batalla):
        self._batalla = batalla
    
    @property
    def batalla(self):
        return self._batalla

    @batalla.setter
    def batalla(self, batalla):
        self._batalla = batalla
    
    def gestionar_batalla(self):
        self.batalla.comenzar_batalla()